Makefile

```diff
@@ -1,30 +1,34 @@
-export CC = clang
 .DEFAULT_GOAL := help
 
-juno: ## compile
+.PHONY: vm
+
+juno: vm ## compile
 	@mkdir -p build
 	@go build -ldflags="-X main.Version=$(shell git describe --tags)" -o build/juno ./cmd/juno/
 
+vm:
+	$(MAKE) -C vm/rust all
```

Comment:
suggestion: since number of make targets slowly increasing wdyt about passing `-j` flag to speed it up a little (I think it would be especially helpful by adding it to make in Dockerfile)

Reply:
I dont think that will help since our make target is just a single command ie `cargo build`. `-j` wouldnt help to parallelize that.